### ISSUE TITLE
bug? raid not working as I'd expect

### ISSUE THREAD

**Issue by bgartzia** [pointsphere/wombot#41]
I don't know if that's how we are supposed to be handling raid, but I met one case that was kind of weird (at least for me).

Typing
```
!raid<whatever>
```

example:
```
!raidmyInnerThoughts 
```
where `<whatever>` is whatever string you wanted just deletes the message. I expected wombot to leave it there as it usually does with unrecognized commands.

**Comment by pointsphere** [comment]
True, it should leave visible anything it doesn't know. IDs need an overhaul anyway, it will be addressed then.

**Comment by bgartzia** [comment]
Message is deleted

https://github.com/pointsphere/wombot/blob/f1db554e97ed922bde689f792843774a65432338/wombot.py#L832-L833

without previously checking whether the station ID is in the supported station ID set or not 🤔

**Comment by pointsphere** [comment]
raid was implemented with no checking beyond "works in principle and doesnt crash".
will be addressed with id overhaul.

**Comment by kktzi** [comment]
Haven't read your comment @ghxm until now sorry. I opened a pull request with a fix draft.
